### ISSUE TITLE
Bold screaming focus outline

### ISSUE THREAD

**Issue by ygoe** [MicrosoftDocs/feedback#886]
**Describe the bug**
Every clicked link has a big fat black dashed outline around it. It's three times the size of a regular keyboard focus rectangle, and also appears when I don't touch my keyboard at all.

**To Reproduce**
1. Click on a link and keep the mouse button pressed.

**Expected behavior**
No outline.

**Screenshots**
Should be visible everywhere, has been confirmed elsewhere for Firefox and Chrome.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version: current stable (was it 92? the number changes every other week)

**Additional context**
If that fat outline is supposed to be a keyboard focus rectangle, then it should only appear when I'm using my keyboard to navigate. Microsoft should be very well aware of what a focus rectangle is. It should not be there when I'm using a pointer input device.

This user style can be used to get rid of most of the outlines. Some `.card` elements have it still though and I failed to analyse where they come from.

```css
.has-default-focus .is-focused,
.has-default-focus:focus:not(.visually-hidden):not(.has-inverted-focus):not(:active),
.has-default-focus :focus:not(.visually-hidden):not(.has-inverted-focus):not(:active),
.lang-toggle-container .checkbox-toggle-wrapper input[type="checkbox"]:focus + label.switch,
.this:focus
{
    outline: none;
}
```

**Comment by Powerhelmsman** [comment]
@EdisonDa Can you take a look at this issue?

**Comment by EdisonDa** [comment]
@ygoe thank you for your reporting. Could you please give me the URL where we can reproduce this issue and the screenshot? It will help us to investigate and fix the issue.

**Comment by EdisonDa** [comment]
@ygoe thank you for your quick turnaround. @ShevaDas looks this is the latest change, could you please confirm it's by design?

**Comment by ygoe** [comment]
Here's a similar issue: https://www.microsoft.com/de-de/p/microsoft-surface-dock/8qrh2npz0s0p Click on a tab ("Übersicht", "Technische Spezifikationen", "Kritiken") and you get a solid 2px black outline. It even stays there indefinitely. Seems Microsoft has found pleasure in emphasising keyboard focus.

**Comment by ShevaDas** [comment]
@ygoe thanks for the report! I've confirmed that the dashed line is how it should look, but it should only appear with keyboard navigation. I've opened an issue with our engineering team and will follow up here when it's fixed.

<internal tracking: https://ceapex.visualstudio.com/Engineering/_workitems/edit/52882>

**Comment by shirgoldbird** [comment]
Hi @ygoe, thanks for your patience while we investigated this further. After discussing with our design partners, we've determined this is the approved Microsoft styling guidance for links and what we'll continue to use on our site. I'm going to close this out—please feel free to open a new issue if you have further feedback on the site.

**Comment by ygoe** [comment]
Alright. I don't like ugly links and design concept misuse, so I'll extend my user style to other sites and continue to use and advertise it to fix this issue on the client side. Thanks for your time, now I'll be using mine.
